test/e2e_node/eviction_test.go

```diff
@@ -725,6 +752,21 @@ func verifyEvictionOrdering(f *framework.Framework, testSpecs []podEvictSpec) er
 	return fmt.Errorf("pods that should be evicted are still running: %#v", pendingPods)
 }
 
+func verifyPodConditions(f *framework.Framework, testSpecs []podEvictSpec) {
+	for _, spec := range testSpecs {
+		if spec.evictionPriority != 0 && spec.wantPodDisruptionCondition != nil {
```

Comment:
why do we need to check `spec.evictionPriority != 0`? Can we just check if  `spec.wantPodDisruptionCondition != nil`?

Reply:
true, simplified